packages/ni-redirects/src/index.js

```diff
@@ -21,20 +27,25 @@ export default {
 	async fetch(request, env) {
 		/** @type {EnvConfig} */
 		const config = env.CONFIG === 'prod' ? prodConfig : testConfig;
+		/** @type {EnvGeneratedConfig} */
+		const generatedConfig = env.CONFIG === 'prod' ? prodGeneratedConfig : testGeneratedConfig;
+		const redirectAllowedCaseReferences = generatedConfig.redirectAllowedCaseReferences;
+		console.log(`Enabling redirection for the following cases: ${redirectAllowedCaseReferences}`);
```

Comment:
Maybe only log in test? This will use up precious CPU cycles!

Reply:
Thanks good shout, changed to only log for test